Refit/VV-Refit.csproj

```diff
@@ -2,7 +2,9 @@
 
   <PropertyGroup>
     <Product>Refit ($(TargetFramework))</Product>
-    <TargetFrameworks>netstandard1.4;net45</TargetFrameworks>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
TargetFramework should not be changed

Reply:
Please ignore that pull request. I made another with all fixed. #414

Reply:
Due some mistakes, I made another pull request with all fixed. #414